Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextToolbar/TextToolbarPage.xaml.cs

```diff
@@ -157,7 +157,7 @@ private void Previewer_LinkClicked(object sender, LinkClickedEventArgs e)
             try
             {
                 var link = new Uri(e.Link);
-                var linkOpen = Task.Run(() => Launcher.LaunchUriAsync(link));
+                _ = Launcher.LaunchUriAsync(link);
```

Comment:
Why discard the result rather than awaiting the call?

Reply:
LaunchUri is just going to go open up the browser for a link from the user markdown here; why would I want to wait on that operation to continue?  Especially since we're not doing anything with the result?

Reply:
Because it's the end of the method and it allows the catch to do it's job

Reply:
Exception is eaten in both cases regardless, the try/catch here is more for malformed Uris than the LaunchUriAsync.